core/ide/src/main/java/org/alice/stageide/sceneeditor/interact/manipulators/Camera2DDragStrafeManipulator.java

```diff
@@ -79,19 +79,19 @@ protected void initializeEventMessages() {
 
   @Override
   protected Vector3 getMovementVectorForColor(Color color) {
-    Vector3 initialMove = new Vector3(0.0d, 0.0d, 0.0d);
-    if (color != null) {
-      if (color.equals(UP)) {
-        initialMove.y = INITIAL_MOVE_FACTOR;
-      } else if (color.equals(DOWN)) {
-        initialMove.y = -INITIAL_MOVE_FACTOR;
-      } else if (color.equals(LEFT)) {
-        initialMove.x = -INITIAL_MOVE_FACTOR;
-      } else if (color.equals(RIGHT)) {
-        initialMove.x = INITIAL_MOVE_FACTOR;
-      }
+    if (UP.equals(color)) {
```

Comment:
you switched from MOVE_FACTOR to ROTATE_FACTOR

Reply:
Oops. Fixed.